docs/en/observability/cloud-monitoring/aws/monitor-amazon-sqs.asciidoc

```diff
@@ -0,0 +1,140 @@
+[[monitor-amazon-sqs]]
+= Monitor Amazon Simple Queue Service (SQS)
+
+++++
+<titleabbrev>Monitor SQS</titleabbrev>
+++++
+
+The main parts in a distributed messaging system include the software components
+of your system, the queue (distributed on Amazon SQS), and the messages in the
+queue.
+
+Amazon Simple Queue Service (SQS) is a message queuing service that makes it
+easier to decouple and scale microservices, distributed systems, and serverless
+applications. 
+
+AWS SQS and Amazon CloudWatch are integrated and therefore, you can collect,
+view, and analyze data. Metrics for Amazon SQS queues are automatically
+collected and pushed to CloudWatch at one-minute intervals.
+
+You can view and analyze the queue’s metrics from the Amazon SQS console, the
+CloudWatch console, the AWS CLI, or by using the CloudWatch API.
+
+After a message is received, it remains in the queue for consumers to ingest.
```

Comment:
@aspacca Can you take a moment to look at this paragraph and let me know if it belongs here. I think it's not clear why the user needs to know this or what they need to do.

Reply:
@kaiyan-sheng can you look at it? thanks

Reply:
@dedemorton This is not related to the metric collection so we need to remove it.